packages/css/src/components/Form/form.css

```diff
@@ -28,6 +29,10 @@
   color: var(--form-placeholder-color);
 }
 
+.form-input:focus {
+  border-color: var(--form-focus-border-color);
+}
+
```

Comment:
For me this is overshadowed by the input outline, which is also bluish

Reply:
Should be fine now.